neard/src/config.rs

```diff
@@ -932,18 +938,23 @@ pub fn get_genesis_url(chain_id: &String) -> String {
 pub fn download_genesis(url: &String, path: &PathBuf) {
     info!(target: "near", "Downloading config file from: {} ...", url);
 
-    let mut file = File::create(path).expect("Failed to create / write a config file.");
-    let mut easy = Easy::new();
+    let url = url.clone().to_string();
```

Comment:
`.clone()` should be enough. If not, it is better to have explicit type annotation instead of `.to_string()`.